packages/deployer/src/utils/evm.ts

```diff
@@ -36,10 +29,12 @@ export function generateMnemonic() {
   return masterWallet.mnemonic.phrase;
 }
 
-export function shortenAirnodeId(airnodeId: string) {
-  logger.debug('Shortening Airnode ID');
-  if (!ethers.utils.isHexString(airnodeId, 32)) {
-    throw new Error('airnodeId is not a valid hex string');
+export function shortenAirnodeAddress(airnodeAddress: string) {
+  logger.debug('Shortening Airnode Address');
+  try {
+    ethers.utils.getAddress(airnodeAddress);
+  } catch {
+    throw new Error('airnodeAddress is not a valid hex string');
   }
-  return airnodeId.substring(2, 9);
+  return airnodeAddress.substring(2, 9);
```

Comment:
@amarthadan You mentioned that this needs to be lowercased. 
But maybe I misunderstood?

Reply:
That is correct. I'm just calling toLowerCase() where I'm calling [this](https://github.com/api3dao/airnode/blob/fa68fc7fe787ca4235d386d17f04a67ea641cfe3/packages/deployer/src/cli/commands.ts#L49) function because I thought that making lower case is AWS specific and this  shortenAirnodeAddress() might be called for other purposes in the future.

Reply:
I'd still move it here instead. Even if it is AWS specific issue I would like to have the lowercased version everywhere, not just there.

Reply:
Ok I'll move it here and I'll apply the same logic in node package and also add it in [here](https://github.com/api3dao/airnode/blob/e0acbf5f3c8fc263e09e32ec818940a7886f3bdb/packages/node/src/evm/wallet.ts#L45).